.github/workflows/ci.yml

```diff
@@ -373,6 +373,14 @@ jobs:
         shell: bash
         run: echo OPENSSL_SRC_PERL=C:/Strawberry/perl/bin/perl >> $GITHUB_ENV
 
+      - name: Additional Check
+        if: startsWith(matrix.os, 'ubuntu')
+        run: |
+          set -ex
+          rustup target add wasm32-unknown-unknown
+          cargo check --no-default-features --features use-sodiumoxide
```

Comment:
What the need for that ? I would say check is basically the first half of compilation, so this line is redundant with when we do `cargo test --profile ci-rust --package libparsec_crypto --no-default-features --features use-sodiumoxide`

Reply:
Here we compile all with sodiumoxide, that's how we can detect if the package will work